src/main/java/seedu/address/logic/commands/edit/DeleteModuleRoleOperation.java

```diff
@@ -37,10 +37,10 @@ protected ModuleRoleMap execute(ModuleRoleMap moduleRoleMapToEdit) throws Comman
         ModuleRoleMap failed = ret.removeAll(moduleRoleMapToDelete);
         if (!failed.isEmpty()) {
             throw new CommandException("You wish to delete these module role pair(s) but they do not exist: "
```

Comment:
I think this could be considered as use of magical literal as well, you can refer to CommandException message in `AddCommand` to extract out the message as variables and make use of `String.format` to construct the error message instead